ceph.yaml

```diff
@@ -0,0 +1,344 @@
+package:
+  name: ceph
+  version: 19.2.1
+  description: Distributed object, block, and file storage
+  copyright:
+    - license: LGPL-2.1
+  dependencies:
+    runtime:
+      - curl
+      - libaio
+      - libedit
+      - libgcc
+      - libstdc++
+      - libudev
+      - openssl
+      - py${{vars.py-version}}-bcrypt-3.2
+      - py${{vars.py-version}}-cherrypy
+      - py${{vars.py-version}}-cryptography
+      - py${{vars.py-version}}-jinja2
+      - py${{vars.py-version}}-jsonpatch
+      - py${{vars.py-version}}-packaging
+      - py${{vars.py-version}}-pecan
+      - py${{vars.py-version}}-prettytable
+      - py${{vars.py-version}}-pyopenssl
+      - py${{vars.py-version}}-python-dateutil
+      - py${{vars.py-version}}-pyyaml
+      - py${{vars.py-version}}-requests
+      - py${{vars.py-version}}-scipy
+      - py${{vars.py-version}}-urllib3
+      - py${{vars.py-version}}-werkzeug
+      - python-${{vars.py-version}}
+      - yaml
+      - zlib
+
+vars:
+  py-version: 3.11
+
+environment:
+  contents:
+    packages:
+      - apache-arrow-dev
+      - autoconf
+      - automake
+      - bash
+      - bison
+      - boost-dev
+      - build-base
+      - cmake
+      - cryptsetup-dev
+      - curl-dev
+      - doxygen
+      - eudev
+      - expat-dev
+      - flex
+      - fuse3-dev
+      - gperf
+      - icu-dev
+      - jq
+      - keyutils-dev
+      - libaio-dev
+      - libarrow
+      - libcap-dev
+      - libcap-ng-dev
+      - libedit-dev
+      - libnl3-dev
+      - librdkafka-dev
+      - libtirpc-dev
+      - libtool
+      - liburing-dev
+      - libxml2-dev
+      - lttng-ust-tools
+      - lua5.4-dev
+      - lz4-dev
+      - make
+      - nasm
+      - ncurses-dev
+      - ninja
+      - nodejs
+      - npm
+      - oath-toolkit-dev
+      - openldap-dev
+      - openssl-dev
+      - pkgconf
+      - posix-libc-utils
+      - py${{vars.py-version}}-cython
+      - py${{vars.py-version}}-prettytable
+      - py${{vars.py-version}}-pyyaml
+      - py${{vars.py-version}}-setuptools
+      - py${{vars.py-version}}-sphinx-bin
+      - python-${{vars.py-version}}-dev
+      - re2-dev
+      - snappy-dev
+      - swagger
+      - systemd-dev
+      - thrift-dev
+      - util-linux-dev
+      - wolfi-base
+      - xfs-scrub
+      - xfsprogs
+      - yaml-cpp-dev
+      - zlib-dev
+      - zstd-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/ceph/ceph.git
+      tag: v${{package.version}}
+      expected-commit: 58a7fab8be0a062d730ad7da874972fd3fba59fb
+      recurse-submodules: true
+
+  - uses: patch
+    with:
+      patches: fix-submodule.patch fix-osd-class-dir.patch
+
+  - uses: cmake/configure
+    with:
+      opts: |
+        -Wno-dev \
```

Comment:
Should this needs to be omitted since we build `-dev` subpkg? https://apk.chaindag.dev/size/apk.cgr.dev/wolfi-presubmit/d61fbf5c0c06c806544e0b787bae1af5693273a8/aarch64/ceph-dev-19.2.1-r0.apk@etag:35aa97ce37ce4d6dabecc81f2e4e8da4